packages/builder/src/components/backend/DataTable/formula.js

```diff
@@ -62,6 +62,12 @@ export function getBindings({
 
     const label = path == null ? column : `${path}.0.${column}`
     const binding = path == null ? `[${column}]` : `[${path}].0.[${column}]`
+
+    let readableBinding = label
+    if (readableBinding.includes(" ")) {
+      readableBinding = `[${readableBinding}]`
```

Comment:
Is this safe? If the binding is something like `Named Repeater.rows.0.a column` will it not lead to `[Named Repeater.rows.0.a column]` suggesting that the entire binding is a constant string, ignoring the dot notation?

Reply:
You are right. Updated it to support dot notation